apps/website/src/app/docs/report/page.tsx

```diff
@@ -0,0 +1,232 @@
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportIcon />"}</pre>
+            </div>
+
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.pass)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.warn)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.alert)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" hadLoadingError={true} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" />
+            </div>
+          </div>
+          <div className="grid grid-cols-6 gap-x-6 py-5">
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportIcon onClick />"}</pre>
+            </div>
+
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                data={getReportExampleData(Rating.pass)}
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                data={getReportExampleData(Rating.warn)}
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                data={getReportExampleData(Rating.alert)}
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                hadLoadingError={true}
+                name="notrab.eth"
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" onClick={() => alert("Clicked")} />
+            </div>
+          </div>
+        </div>
+
+        <h1 className="justify-center flex font-bold text-2xl mb-8 pt-12">
+          {"<"}ReportBadge {"/>"} documentation
+        </h1>
+
+        <div className="divide-y divide-gray-100">
+          <div className="grid grid-cols-6 gap-x-6 py-5 text-center font-medium">
+            <div></div>
+            <div>Success</div>
+            <div>Warn</div>
+            <div>Alert</div>
+            <div>Unknown</div>
+            <div>Loading</div>
+          </div>
+        </div>
+
+        <div className="grid grid-cols-6 gap-x-6 py-5">
+          <div className="flex items-center">
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportBadge />"}</pre>
```

Comment:
Could you please also create a variation with a very long name for us to test with? For example, please consider this case from Figma:

![CleanShot 2024-05-07 at 17 49 15](https://github.com/namehash/nameguard/assets/126201998/93fe0823-2ab3-49d5-a71d-b3f6072226d5)

https://www.figma.com/file/Kt4nUHyZJ5NuPCSZZkXmPL/NameGuard?type=design&node-id=390-3364&mode=design&t=wrzBFLo8iSoUN5bc-4